Lib/test/test_future_stmt/test_future.py

```diff
@@ -25,57 +25,71 @@ def check_syntax_error(self, err, basename, lineno, offset=1):
         self.assertEqual(err.offset, offset)
 
     def test_future1(self):
```

Comment:
test_future1 has an interesting comment that you can copy there:

```
# Import the name nested_scopes twice to trigger SF bug #407394 (regression).
```

I suggest to rename the file and test to: ``test_import_nested_scope_twice``. For the filename, just drop ``test_`` prefix: ``import_nested_scope_twice.py``?